react/features/base/conference/actions.js

```diff
@@ -106,7 +106,14 @@ function _addConferenceListeners(conference, dispatch, state) {
         (...args) => dispatch(conferenceFailed(conference, ...args)));
     conference.on(
         JitsiConferenceEvents.CONFERENCE_JOINED,
-        (...args) => dispatch(conferenceJoined(conference, ...args)));
+        (...args) => {
```

Comment:
could we keep the code format that's already used? or was there a reason for changing it?

Reply:
You mean the one from web?

Reply:
Its the same, but I just do not add the the sessionId property the meeting unique id

Reply:
ah, I see what you mean

Reply:
It remained like this because of debugging, I'll change them now